pcp/versions/next/unauthorized.md

```diff
@@ -0,0 +1,50 @@
+Unauthorized
+===
+
+When a broker declines to deliver a message because it fails authorization,
+it must send an *unauthorized* message back to the original message sender.
+
+A clinet may send a message to multiple recipients while it's only authorized
```

Comment:
The hypothetical form is weird when you're introducing a figure. Also, after the above paragraph, it's not clear that you're actually moving to an example.

Something like "Let's consider the the figure below, where client A tries to send a message to ..." may sound more clear.